generator-prisma-client/deno-deploy/README.md

```
@@ -63,26 +63,26 @@ Note that `__YOUR_PRISMA_POSTGRES_CONNECTION_STRING__` is a placeholder value th
 
-The [Prisma schema file](./prisma/schema.prisma) contains a single `Quotes` model and a `QuoteKind` enum. You can map this model to the database and create the corresponding `Quotes` table using the following command:
+Run:
 
 ```
-deno task prisma migrate dev --name init
+deno task prisma generate
 ```
 
-You now have an empty `Quotes` table in your database. Next, run the [seed script](./prisma/seed.ts) to create some sample records in the table:
+### 4. Run a migration to create the database structure and seed the database
+
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)